### DISCUSSION TITLE
Extra network tab UI

### DISCUSSION THREAD

**Discussion by clacheek** [vladmandic/sdnext#1631] [Q&A] [answered]
does someone knows how to make this folder tab wider? cause it's useless at this point... I did tweaked some settings in extra network tab but none seems to help _(the older extra tab was bad but not as bad as this new one imo)_
![image](https://github.com/vladmandic/automatic/assets/114445644/49d40469-0e9a-452d-bd29-f1661a95ebb2)

**Comment by vladmandic** [comment, marked as answer]
I'll make it wider

**Reply by clacheek** [reply]
Is there any way that I can go back to the old Extra network UI thumbnail system thing? This new x.thumb.jpg is making my extra nets folders a mess, before that just having one .png alongside each safetensors files was enough to display thumbs on the webui tab.

**Reply by vladmandic** [reply]
How is one file per model making a mess??

**Reply by clacheek** [reply]
just answear my question, there is a way to go back to the old thumb system?

**Reply by brunogcar** [reply]
yea sure, just go back to commit right before the change and never update again,
but if you are asking if is there an option to select old x new Extra network UI,  I wouldn't recommend wait for it,  cos that's not going to happen

**Reply by vladmandic** [reply]
> just answear my question, there is a way to go back to the old thumb system?

open source and free -> you're not going to get far with that attitude.

**Reply by clacheek** [reply]
sorry for my behave, i was really pissed with this thing
